packages/experiments/src/components/CollapsibleSection/CollapsibleSection.view.tsx

```diff
@@ -23,16 +24,71 @@ export const CollapsibleSectionView = (
 ) => {
   const { collapsed, titleAs: TitleType = CollapsibleSectionTitle, titleProps, children } = props;
 
-  // TODO: make sure onToggleCollapse gets called both from state and from titleProps.
+  const onRootKeyDown = (ev: React.KeyboardEvent<Element>) => {
+    const rootKey = getRTL() ? KeyCodes.right : KeyCodes.left;
+    switch (ev.which) {
+      case rootKey:
+        if (props.titleElementRef && props.titleElementRef.value && ev.target !== props.titleElementRef.value) {
```

Comment:
.value is deprecated, use .current. See React docs. https://reactjs.org/docs/refs-and-the-dom.html

Also perhaps in experiments we can remove "deprecations" tslint exclusion to catch this (in some other pr!)

Reply:
Deprecations was added not for this, but for React.Props usage which OUFR also has all over. I'll change this, though.